Nodejs/Product/Npm/SemverVersion.cs

```diff
@@ -31,13 +31,13 @@ public struct SemverVersion {
             "^(?<major>[0-9]+)"
             + "\\.(?<minor>[0-9]+)"
             + "\\.(?<patch>[…0-9]+)" // The '…' is there to handle the 'classy' library, which has a very long version number - see slightly snarky comment about that unadulterated bag of hilarity below.

```

Comment:
Ah, got it. So it's not part of the version number, just part of the textual output the code used to scrape, where it used this char when truncating a column. Makes sense.

It would be good to just delete these if no longer needed. I spent quite a while looking at the SemVer spec, the 'classy' history, and the RegEx operators to trying and figure out what the hell it might be doing :confused: